app/assets/stylesheets/mixin.scss

```diff
@@ -4,21 +4,7 @@
 }
 
 @mixin linear_gradient($start_color, $end_color) {
-  background: $start_color; /* Old browsers */
-  background: -moz-linear-gradient(top, $start_color 0%, $end_color 100%); /* FF3.6+ */
-  background: -webkit-gradient(linear, left top, left bottom, color-stop(0%,$start_color), color-stop(100%,$end_color)); /* Chrome,Safari4+ */
-  background: -webkit-linear-gradient(top, $start_color 0%,$end_color 100%); /* Chrome10+,Safari5.1+ */
-  background: -o-linear-gradient(top, $start_color 0%,$end_color 100%); /* Opera 11.10+ */
-  background: -ms-linear-gradient(top, $start_color 0%,$end_color 100%); /* IE10+ */
-  background: linear-gradient(to bottom, $start_color 0%,$end_color 100%); /* W3C */
-  filter: progid:DXImageTransform.Microsoft.Gradient(GradientType=0, startColorstr='#{$start_color}', endColorstr='#{$end_color}');
-  -ms-filter: "progid:DXImageTransform.Microsoft.gradient(startColorstr='#{$start_color}', endColorstr='#{$end_color}')";
-}
-
-@mixin border_radius($top-right, $top-left, $bottom-left, $bottom-right){
-  -webkit-border-radius: $top-right $top-left $bottom-left $bottom-right;
-  -moz-border-radius: $top-right $top-left $bottom-left $bottom-right;
-  -ms-border-radius: $top-right $top-left $bottom-left $bottom-right;
-  -o-border-radius: $top-right $top-left $bottom-left $bottom-right;
-  border-radius: $top-right $top-left $bottom-left $bottom-right;
+  background: $start_color; /* fallback for older browsers */
```

Comment:
`border-radius` now works unprefixed on 89.44% of all browsers (by market share), adding vendor prefixes only add 0.22% support - and even that is only in ancient browsers which are likely not used by our user base anyways. Therefor I think in favor of lesser code complexity there is no need for a mix-in. (Data about browser support and share from [here](http://caniuse.com/#feat=border-radius))